docs/conf/australia/2024/sponsors/prospectus.rst

```diff
@@ -22,14 +22,18 @@ but please let us know if there are other points of interaction with our communi
 Concept
 =======
 
-Write the Docs Australia (http://www.writethedocs.org/) is a **two day conference** focusing on documentation systems, tech writing
-theory, and information delivery.
-
-Writing and maintaining documentation involves a multidisciplinary community of technical writers, designers, librarians, typesetters, developers,
-support teams, marketers and many others. This group of people can be collectively referred to as “documentarians”.
-
-**Write the Docs** creates a time and a place for this community to share information, discuss ideas, and work together to improve the art
-and science of documentation.
+Write the Docs Australia (http://www.writethedocs.org/) is a **two day conference** dedicated to connecting content 
+professionals from various disciplines. Our community includes technical writers, programmers, support teams, developer 
+advocates, content strategists and more. 
+We collectively refer to ourselves as “`documentarians <https://www.writethedocs.org/documentarians/>`_”. 
+
+Each year, we 
+invite community members to share their insights and experiences. Between the talks on stage, discussions in the unconference
```

Comment:
question: wonder if "invite" would imply invited guests as opposed to the CFP process. Maybe use "encourage" to avoid confusion?

Reply:
Yes! Nice catch.